frontend-cert/js-projects/lab-lightbox/user-stories.md

```diff
@@ -0,0 +1,25 @@
+A lightbox displays a larger version of an image when clicked and shadows the rest of the page. This project will create a lightbox gallery that displays full-size images when a thumbnail is clicked. For each image, two versions are provided: a thumbnail and a full-size image. The full-size image is the same as the thumbnail, but without the `-thumbnail` suffix.
+
+1. You should have a `div` with a class of `gallery` within your `body`.
+2. Within the `.gallery` element, you should have at three image thumbnails, each with a class of `gallery-item`. You should use the following links for thumbnail images: 
```

Comment:
```suggestion
2. Within the `.gallery` element, you should have three image thumbnails, each with a class of `gallery-item`. You should use the following links for thumbnail images: 
```

I think if we're going to use these images, we can just make it so they only add the three?